distro/Makefile

```diff
@@ -37,12 +44,16 @@ get-artifacts: dist
 	# get cli artifacts
 	cd .. && mvn clean package && cd -
 	cp ../target/hazelcast-command-line-*-jar-with-dependencies.jar ${HZ_LIB}/hazelcast-command-line.jar
-	cp src/artifacts/hazelcast-management-center-*.war ${HZ_ARTIFACTS}
+	wget -O ${MC_INSTALL_TAR} http://download.hazelcast.com/management-center/${MC_INSTALL_TAR}
+	tar -zxf ${MC_INSTALL_TAR}
+	rm -rf ${MC_INSTALL_TAR}
+	mv ${MC_INSTALL_NAME}/${MC_INSTALL_JAR} ${HZ_ARTIFACTS}
+	rm -rf ${MC_INSTALL_NAME}
```

Comment:
Just like above?

```suggestion
	rm ${MC_INSTALL_NAME}
```

Reply:
This one is the extracted folder, thus it is correct.

Reply:
Oh, right 👍